client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -26,7 +26,14 @@ export default React.createClass( {
 		icon: React.PropTypes.string,
 		event: React.PropTypes.string,
 		href: React.PropTypes.string,
-		jetpack: React.PropTypes.bool
+		jetpack: React.PropTypes.bool,
+		feature: React.PropTypes.oneOf( [ 
+			false,
```

Comment:
Yeah, I agree... we should have a separate constants filefor active nudges, whose keys can be used for passing the nudge data through steps as well as for analytics.